cli/client.go

```diff
@@ -232,6 +232,11 @@ var clientDealCmd = &cli.Command{
 			Usage: "specify the epoch that the deal should start at",
 			Value: -1,
 		},
+		&cli.BoolFlag{
+			Name:  "fast-retrieval",
+			Usage: "indicates that data should be available for fast retrieval",
+			Value: true,
```

Comment:
This should default to false, otherwise non-verified clients making deals will have their deals rejected unless the explicitly say they arent verified here

Reply:
That's the fast-retrieval flag, and I think it's ok to default to true (we still want miners to provide fast-ret deals to non-verified clients)